### PR TITLE
Add a hover effect to the tab menu of ACF page

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #425 

## Goals

Adding a hover effect to the links in the top of the acf page

## Approach
 
I added a hover effect to the links in the top of the acf page

### Screenshots

image

  
### Preview Link

https://pr-427-sack-site-acf.surge.sh/acf

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/css/custom.css [hunk 1]
```diff
@@ -456,6 +456,10 @@
   transition: 0.3s background-color 0s;
 }
 
+.society-nav-link:hover, .society-nav-link.active:hover {
```
Comment (Piumal1999) [inline assets/css/custom.css:459]: I think we already have a custom css class for that. 
Try using `nav-btn-hover` class with the tab. 

Example:
```
<li class="nav-item">
    <a class="society-nav-link nav-btn-hover" href="#google-classroom" data-toggle="tab">
        Google Classroom
    </a>
</li>
```

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

File: assets/css/custom.css (hunk 1 above)
Comment (YoshithaRathnayake) [reply assets/css/custom.css]: Ok @YoshithaRathnayake done!

File: acf/index.html [hunk 2]
```diff
@@ -13,25 +13,25 @@
...
+							<a class="society-nav-link nav-btn-hover" href="#google-classroom" data-toggle="tab">
 								Google Classroom
 								<div class="ripple-container"></div>
 							</a>
 						</li>
 						<li class="nav-item">
-							<a class="society-nav-link" href="#advisor-panel" data-toggle="tab">
+							<a class="society-nav-link nav-btn-hover" href="#advisor-panel" data-toggle="tab">
 								Advisor Panel
 								<div class="ripple-container"></div>
```
Comment (Piumal1999) [inline acf/index.html:30]: I think we can remove these ripple containers also. The ripple effect automatically comes with the links

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

File: acf/index.html (hunk 2 above)
Comment (YoshithaRathnayake) [reply acf/index.html]: Ok done @Piumal1999 !

**Review by Piumal1999 — APPROVED** [review]
Good job @YoshithaRathnayake 
Thank you!

**Pull request merged**